common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3389,12 +3389,63 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_TABLE_OPERATION" : {
+    "message" : [
+      "The table <tableName> does not support <operation>."
+    ],
+    "subClass" : {
+      "WITHOUT_SUGGESTION" : {
+        "message" : [
+          ""
+        ]
+      },
+      "WITH_SUGGESTION" : {
+        "message" : [
+          "Please use ALTER TABLE instead."
+        ]
+      }
+    }
+  },
+  "UNSUPPORTED_TEMP_VIEW_OPERATION" : {
```

Comment:
Can we remove this? temp view is also view and there is nothing wrong to say `The view abc does not ...` even if `abc` is a temp view